src/FunctionApps/python/tests/IntakePipeline/test_pipeline.py

```diff
@@ -78,18 +87,16 @@ def test_pipeline_valid_message(
     patched_name_standardization.assert_called_with(
         {"resourceType": "Bundle", "entry": [{"hello": "world"}]}
     )
-    patched_phone_standardization.assert_called_with(
-        {"resourceType": "Bundle", "entry": [{"hello": "world"}]}
-    )
+    patched_phone_standardization.assert_called_with(patched_standardized_name_data)
```

Comment:
No, I think this is correct as written. If you look at lines 62 to 72, you'll see that the objects named `X_data` are the return values of the transform that standardizes `X`. So, `patched_phone_standardization` as a mock will generate the returned data `standardized_phone_data`. The actual phone standardization itself is run on the _name_ standardized data, which means this assertion is correct because `patched_phone_standardization` _should_ be called on the result of `patched_name_standardiation` and not on its own resulting data.